hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkDeletePartitionCommitActionExecutor.java

```diff
@@ -18,13 +18,17 @@
 
 package org.apache.hudi.table.action.commit;
 
+import java.util.ArrayList;
```

Comment:
Please move after line 44 and follow the import check style.

Reply:
Done!